### PR TITLE
feat(portex): add method to convert Portex schema to Avro schema

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2c1yc99 修复 Portex schema 的 enum 转化为 avro schema 时变为 string 的问题](https://app.clickup.com/t/2c1yc99)

File: graviti/utility/__init__.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 from graviti.utility.attr import AttrDict
 
 # https://github.com/python/mypy/issues/9318
```
Comment (graczhual) [inline graviti/utility/__init__.py:9]: Do we still need this line？

File: graviti/portex/avro.py [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+def _on_list(names, namespace, name, _pa_ist: array) -> AvroArraySchema:
+    sub_namespace = f"{namespace}.{name}.items"
+    sub_name = "items"
+    items = _on_type(names, sub_namespace, sub_name, 0, _pa_ist.items)
+    return AvroArraySchema(items=items)
+
+
+def _on_primitive(portex_type: PortexType) -> AvroSchema:
+    if isinstance(portex_type, (int32, int64)):
```
Comment (linjiX) [inline graviti/portex/avro.py:160]: use `dict`?

File: graviti/portex/builtin.py [hunk 3]
```diff
@@ -477,6 +477,8 @@ class tensor(PortexBuiltinType):  # pylint: disable=invalid-name
 
     """
 
+    _T = TypeVar("_T", bound="tensor")
```
Comment (linjiX) [inline graviti/portex/builtin.py:480]: what is this ?

File: graviti/utility/__init__.py [hunk 4]
```diff
@@ -57,7 +54,6 @@
     "UserSequence",
     "chunked",
     "config",
-    "convert_arrow_schema_to_avro",
```
Comment (linjiX) [inline graviti/utility/__init__.py:60]: shall we remove the old `avro.py`?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by linjiX — APPROVED** [review]

**Review by graczhual — APPROVED** [review]

**Pull request merged**